android/quest/src/test/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModelTest.kt

```diff
@@ -200,10 +201,25 @@ class ProfileViewModelTest : RobolectricTest() {
     every { listActionConfig.handleClickEvent(navController, resourceData) } just runs
     profileViewModel.onEvent(event)
     every { mockProfileViewModel.onEvent(any()) } just runs
-
     verifyAll {
       mockActionConfig.interpolate(any())
       profileViewModel.onEvent(event)
     }
   }
+
+  @Test
+  fun testThatChangeManagingEntityDisplaysBottomSheetWhenEligibleEntitiesExist() {
```

Comment:
```suggestion
  fun testChangeManagingEntityDisplaysBottomSheetWhenResourceTypeIsGroup() {
```

Just an example, feel free to re-word it, for better clarity on what makes it eligible.

Reply:
Yes sure working on it